graphql/src/Mu/GraphQL/Server.hs

```diff
@@ -45,10 +45,10 @@ import qualified Data.HashMap.Strict              as HM
 import           Data.Proxy                       (Proxy (..))
 import qualified Data.Text                        as T
 import           Data.Text.Encoding               (decodeUtf8')
+import           Data.Text.Encoding.Error         (UnicodeException (..))
```

Comment:
why do you need this change in this PR?

Reply:
It was a leftover from my previous PR, my OCD told me that the imports needed to be alphabetised 🤪